src/utils/LogsUtil.ts

```diff
@@ -385,34 +372,35 @@ export async function deleteLogDay(unix_date: number) {
         window.showInformationMessage("Currently waiting to delete the requested log, please wait.");
         return;
     }
-    const jwt = getItem("jwt");
-    if (jwt) {
-        currently_deleting_log_date = unix_date;
-        const resp = await softwareDelete("/100doc/logs", { unix_dates: [unix_date] }, jwt);
-        if (isResponseOk(resp)) {
-            window.showInformationMessage("Your log has been successfully deleted.");
-            // delete the log
-            let logs: Array<Log> = await getLocalLogsFromFile();
-            // delete the log based on the dayNum
-            logs = logs.filter((n: Log) => n.unix_date !== unix_date);
-            saveLogsToFile(logs);
-            await syncLogs();
-            commands.executeCommand("DoC.viewLogs");
-        }
-        currently_deleting_log_date = -1;
+
+    currently_deleting_log_date = unix_date;
```

Comment:
Will this delete all logs on this unix date?

Reply:
oh, I haven't opened up the delete logic. I think I was just getting that ready. I haven't thought about the delete but now that we have unix_date as an exact value, we can use that.